core/app/modules/kit-library/assets/js/pages/favorites/favorites.js

```diff
@@ -6,7 +6,7 @@ export default function Favorites( props ) {
 	const navigate = useNavigate();
 
 	const indexNotResultsFavorites = <ErrorScreen
-		title={ __( 'No favorites here yet...', 'elementor' ) }
+		title={ __( 'No favorites here yet…', 'elementor' ) }
```

Comment:
All ellipses are not worth the re-translating.

Reply:
https://github.com/elementor/elementor/pull/18596/commits/74236ae5d9537474d4854e23709854cc09cf89af